llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -1150,6 +1150,7 @@ class VPseudoUnaryNoMaskGPROut :
 class VPseudoUnaryMaskGPROut :
       Pseudo<(outs GPR:$rd),
              (ins VR:$rs1, VMaskOp:$vm, AVL:$vl, sew:$sew), []>,
+      RISCVMaskedPseudo<MaskIdx=1>,
```

Comment:
Nit, add instead of adding it in the class move it to the two `def`s so it's consistent with other uses of RISCVMaskedPseudo?